core/renderers/zelos/constants.js

```diff
@@ -303,6 +303,13 @@ Blockly.zelos.ConstantProvider = function() {
    */
   this.FIELD_DROPDOWN_SVG_ARROW = true;
 
+  /**
+   * A dropdown field's SVG arrow padding.
+   * @type {number}
+   * @const
+   */
+  this.FIELD_DROPDOWN_SVG_ARROW_PADDING = this.FIELD_BORDER_RECT_X_PADDING;
```

Comment:
Since this constant is used in core/field_dropdown, not only zelos, I think you need to add this constant to common/constants.js

Reply:
Fixed.